job_spec/INSAR_ISCE_TEST.yml

```
@@ -37,6 +37,18 @@ INSAR_ISCE_TEST:
           - HRRR
           - HRES
           - GMAO
+    do_esd:
+      api_schema:
+        description: Whether or not to perform Enhanced Spectral Diversity for improved coregistration
+        defult: false
+        type: boolean
+    esd_coherence_threshold:
+      api_schema:
+        description: Coherence threshold for points to include when performing ESD
+        defult: 0.5
```

```suggestion
        defult: 0.85
```
To reflect default in ISCE2: https://github.com/isce-framework/isce2/blob/main/applications/topsApp.py#L238